### PR TITLE
Rename Logs API to Logs Bridge API to prevent confusion

### PR DESCRIPTION
**Pull request by tigrannajaryan**
We keep seeing confusion about what Log API is. There was a proposal [1] to use the term Bridge API to help prevent the confusion.

Please take a look and comment on whether you think this renaming helps and is reasonable.

[1] https://github.com/open-telemetry/opentelemetry-specification/pull/3187#issuecomment-1424431424

### CODE REVIEW

**Review by reyang — APPROVED** [review]

**Review by jack-berg — APPROVED** [review]

**Comment by yurishkuro** [comment]
Not a fan of the name Bridge for this. It conflicts with the use of this term in OpenTracing/OpenCensus Bridge, and it's not immediately apparent what it even means, i.e. it could easily mean the opposite: User -> Bridge -> language logging API.

Why not use the term we already use - Log Exporter API?

**Comment by Aneurysm9** [comment]
I think this is very analogous to the existing bridges we provide for OpenTracing and OpenCensus, but for logging libraries like log4j or logrus.  It provides a way for existing uses of those libraries to integrate with OTel without requiring changes to instrumentation code.

**Comment by tigrannajaryan** [comment]
> Why not use the term we already use - Log Exporter API?

Log Exporter _interfaces_ is already part of Log SDK. I think naming this document a Log Exporter API would be confusing.

**Comment by jkwatson** [comment]
big fan!

**Review by trask — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
@open-telemetry/specs-logs-approvers please review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,9 @@ release.
 
 ### Logs
 
+- Rename Log API to Log Bridge API to prevent confusion.
```
Comment (arminru) [inline CHANGELOG.md:18]: "Log Bridge API" or "Logs Bridge API"?

Comment (arminru) [reply CHANGELOG.md:18]: Same for the image below. The other places say "Logs".

**Review by arminru — APPROVED** [review]

**Review by djaglowski — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (tigrannajaryan) [reply CHANGELOG.md]: Should be plural everywhere. Fixed.

**Review by MSNev — APPROVED** [review]

**Comment by jmacd** [comment]
Have a look at consistent naming with the [MetricProducer](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricproducer) which is similar.

**Review by jsuereth — APPROVED** [review]

**Review by carlosalberto — APPROVED** [review]

**Comment by tigrannajaryan** [comment]
We have a Log SIG meeting today. I will run the "Producer" suggestion with the SIG. Let's keep this open for 1-2 more days.

**Comment by jack-berg** [comment]
I prefer bridge to producer - somehow bridge conveys more clearly that this isn't meant to be used by end users. A user might think it's appropriate for them to be a log producer, but less so that they should be writing a log bridge. 

I think MetricProducer might benefit from changing to MetricBridge to help avoid users accidentally user the producer / bridge interface instead of async instruments.

**Review by zenmoto — APPROVED** [review]
I think "bridge" makes the intent much clearer.

**Comment by tigrannajaryan** [comment]
Discussed this in Log SIG today and there was not much support for the name "Producer API". If anyone strongly believes that "Producer API" is a better name for the Logs API please open a PR and link from here.

I will wait for another day to give that alternate a chance to be proposed (and if proposed and we see support for it we can extend the waiting period).

If no alternate PR is created within a day I am going to merge this PR.

@open-telemetry/specs-approvers last chance to object.

**Pull request merged**